llvm/lib/Target/PowerPC/PPCISelLowering.cpp

```diff
@@ -426,6 +426,7 @@ PPCTargetLowering::PPCTargetLowering(const PPCTargetMachine &TM,
     setLoadExtAction(ISD::EXTLOAD, MVT::f64, MVT::f32, Expand);
 
   setOperationAction(ISD::GET_ROUNDING, MVT::i32, Custom);
+  setOperationAction(ISD::SET_ROUNDING, MVT::Other, Custom);
```

Comment:
This won't work for SPE, since SPE doesn't have standard floating point registers.  The equivalent is a move to the SPEFSCR SPR.